vortexasdk/version.py

```diff
@@ -1 +1 @@
-__version__ = "0.28.0"
+__version__ = "0.28.1"
```

Comment:
For my benefit: this is irrelevant to dependabot right?

Reply:
Yes - this is required to deploy a new version of the SDK & documentation